decidim/decidim
# Change the Elections' setup minimum hours before start setting to 1 hour

#### :tophat: What? Why?

When working with an Election, by default you need to set it up 3 hours before it starts. This means that every single time that you need to try one Election you need to do it 3 hours before. 

This PR changes this to 1 hour, as it makes sense to development and also to production/staging instances. Mind that this is an Env Var that can be changed in the different installations, and in a real-world Election this actually should be done a couple days before. 

 

#### Testing

1. Create a new development_app
2. Sign in as admin
3. Go to an Election inside of a Participatory Space (ie an Assembly)
4. Go to the "Manage steps" screen
5. See that the check is "The setup is being done at least 1 hours before the election starts."

### :camera: Screenshots

!["Manage steps" page screenshot](https://github.com/decidim/decidim/assets/717367/614d6250-9234-4e46-9e71-8f12e3aae258)


:hearts: Thank you!
